mscore/note_groups.ui

```diff
@@ -155,12 +139,59 @@
        </property>
        <property name="sizeHint" stdset="0">
         <size>
-         <width>40</width>
-         <height>20</height>
+         <width>0</width>
+         <height>0</height>
```

Comment:
This is just the spacer.  I don't see why it needs a sizeHint in the first place.  As it is, the QGroupBox has a fixed width and height.